src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -142,12 +142,13 @@ internal static async Task ConfigurePcs(
 
         builder.Services.RegisterBuildAssetRegistry(builder.Configuration);
         builder.AddWorkItemQueues(azureCredential, waitForInitialization: initializeService);
-        builder.Services.AddWorkItemProcessors();
         builder.AddVmrRegistrations(gitHubToken);
         builder.AddMaestroApiClient(managedIdentityId);
         builder.AddGitHubClientFactory();
         builder.Services.AddGitHubTokenProvider();
         builder.Services.AddScoped<IRemoteFactory, DarcRemoteFactory>();
+        builder.Services.AddTransient<CodeFlowWorkItemProcessor>();
```

Comment:
should we also register this in the method below?

Reply:
Maybe we should do `TryRegister` but we should generally give people flexibility to register the processor how they please